README.rst

```diff
@@ -3,26 +3,32 @@
 .. image:: https://circleci.com/gh/trustlines-protocol/relay.svg?style=svg
     :target: https://circleci.com/gh/trustlines-protocol/relay
 
-trustlines relay server
+Trustlines relay server
 =======================
-This documents describes how to install the trustlines relay server
-for local development.
+Introduction
+------------
+The relay server is a component in between the Trustlines clientlib, and deployed
+Trustlines contracts on a chosen blockchain.
 
-If you're trying to setup a complete trustlines system, please visit
-`Running the trustlines system
-<https://github.com/trustlines-protocol/relay/blob/master/docs/RelayServer.md>`__
+It requires a blockchain node to interact with. It also uses a `py-eth-index
+<https://github.com/trustlines-protocol/py-eth-index>`__
+instance that index events that the relay will be able to process and serve to its users.
 
-The relay server provides a REST API. Please visit the `REST API
-Documentation
-<https://github.com/trustlines-protocol/relay/blob/master/docs/RelayAPI.md>`__
-page for more information.
+The main objectives of the relay include:
 
+- Computing a graph of the trustlines in a currency networks
+- Computing the path of mediators for trustlines transfer in this graph
+- Providing an api to get a user events regarding trustlines
+- Deploying identity contracts for users
+- Relaying transactions of the user to a blockchain node
+- Paying for meta-transactions of a user in exchange for a fee
+
+To have more information about trustlines in general, visit the `Trustlines Foundation website
+<https://trustlines.foundation/>`__
```

Comment:
I guess we should rather link to trustlines.network

Reply:
I felt weird linking to trustlines.network from the foundation repo, but I can